src/generic/json_scanner.h

```diff
@@ -46,8 +46,13 @@ struct json_block {
  * strings. When we're done, json_block will fuse the two together by masking out tokens that are
  * part of a string.
  */
+/// Os3
 class json_scanner {
 public:
+  json_scanner()
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR. Remove the /// Os3 comment.

Reply:
done